linajea/tracking/non_minimal_track.py

```diff
@@ -18,10 +18,10 @@ def nm_track(graph, config, selected_key, frame_key='t', frames=None):
 
             The candidate graph to extract tracks from
 
-        config (``TrackingConfig``)
+        config (``SolveConfig``)
 
             Configuration object to be used. The parameters to use when
-            optimizing the tracking ILP are at config.solve.parameters
+            optimizing the tracking ILP are at config.parameters
```

Comment:
I am not sure if this might not be a bit confusing, everywhere else config is the whole configuration object.
And if at some point we need to access something else in the config we have to change it again.

Reply:
I'm okay to change it back, as long as I can test this function without creating a whole TrackingConfig. Perhaps I could make a "TestTrackingConfig" locally in the test file that allows construction with only a SolveConfig. I'll try that now

Reply:
Okay I did it and it was super easy, I probably should have thought of that first. To keep the history consistent (but ugly), I added new commits that reverted the target commits, and then a few to fix a few lines that weren't consistent after the reversion. Of the 5 new commits, the first 4 are all just undoing the change. If we want to rewrite the history and force push before merging, I'm happy to do that, just let me know.